### PR TITLE
Create User Profile Page

### PR DESCRIPTION
**Pull request by SaurabhK05**

### CODE REVIEW

File: src/Pages/Profile/UserActivityData.json
Comment (RyuzaKiRa) [inline src/Pages/Profile/UserActivityData.json:1]: Can we use an Enum here instead of json file?

Comment (hritiksingh13) [inline src/Pages/Profile/UserActivityData.json:1]: Can we use an Enum here instead of json file?

**Review by hritiksingh13 — APPROVED** [review]
Other than the mentioned comment, everything LGTM.

File: src/Pages/Profile/UserActivityData.json
Comment (SaurabhK05) [reply src/Pages/Profile/UserActivityData.json:1]: Yeah, we can

**Pull request merged**